widgets/contact/contact.php

```diff
@@ -913,8 +913,8 @@ function render_form_label( $field_id, $label, $position ) {
 	 * Ajax action handler to send the form
 	 */
 	function contact_form_action( $instance, $storage_hash ) {
-		if ( ! wp_verify_nonce( $_POST['_wpnonce'] ) ) {
-			exit();
+		if ( ! wp_verify_nonce( $_POST['_wpnonce'], '_contact_form_submit' ) ) {
+			return false;
```

Comment:
Is there a reason why you `return false` here rather than using `wp_die()` like everywhere else?